package.json

```diff
@@ -41,5 +41,9 @@
       "pre-commit": "npm run lint",
       "pre-push": "npm run lint && npm test"
     }
+  },
+  "dependencies": {
+    "@material-ui/core": "^4.10.1",
```

Comment:
I don't think these need to be added here.

Reply:
done